### ISSUE TITLE
`create snapshot` error

### ISSUE THREAD

**Issue by Amber1990Zhang** [vesoft-inc/nebula#1522]
I'm using the same code. But in docker, `CREATE SNAPSHOT` failed. It's ok in my virtual machine.
This is how it looks it docker:
```
(user@127.0.0.1) [(none)]> CREATE snapshot
[ERROR (-8)]: Unknown code -32
```
Moreover, inserting data failed, too.

```
(user@127.0.0.1) [gods]> INSERT VERTEX location(name) VALUES hash('test'):('test')
[ERROR (-8)]: Internal Error
(user@127.0.0.1) [gods]> INSERT VERTEX location(name) VALUES hash('test'):('test')
[ERROR (-8)]: Internal Error
```
Docker code version: `b92e757`

**Comment by Amber1990Zhang** [comment]
will add this as a test case

**Comment by Shylock-Hg** [comment]
Seems caused for not return the kvstore error code, I.E. LEADER_CHANGED.

**Comment by bright-starry-sky** [comment]
I've analyzed it with @Amber1990Zhang  at yesterday. `storaged` disconnected caused all errors of storage layer actions  in docker . the same code works well on virtual machine.

**Comment by Shylock-Hg** [comment]
> I've analyzed it with @Amber1990Zhang at yesterday. `storaged` disconnected caused all errors of storage layer actions in docker . the same code works well on virtual machine.

Yes, but I think this doSyncPut don't return the error code to MetaClient is a error too.

**Comment by NicolaCage** [comment]
Nebula standby time is too long, perform insert `ErrMsg: Internal Error, ErrCode: E_EXECUTION_ERROR`
